config

```diff
@@ -177,4 +195,4 @@
 
 # When returning a free-busy report, limit the number of returned
 # occurences per event to prevent DOS attacks.
-#max_freebusy_occurrence = 10000
```

Comment:
please restore original, potentially line feed lost or is missing

Reply:
also done